src/main/java/ru/hogwarts/school/controller/FacultyController.java

```diff
@@ -51,5 +52,10 @@ public ResponseEntity<Collection<Faculty>> findFaculties(@RequestParam(required
         }
         return ResponseEntity.ok(Collections.emptyList());
     }
+    @GetMapping
```

Comment:
```suggestion
    @GetMapping("/by-faculty/{facultyId}")
```
Такой маппинг уже есть, поэтому нужно прописать другой. Например, такой: